index.js

```diff
@@ -153,7 +154,7 @@ bot.on('message', msg => {
 			if (ret) {
 				msg.channel.send('Quote added!')
 			} else {
-				msg.channel.send('Error: incorrect format!\nFormat: `!quoteadd "<quote text>" -<name>`')
+        msg.channel.send('Error: incorrect format!\nFormat: `!quoteadd "<quote text>" -<name>`')
```

Comment:
The web UI definitely got me here.

After merging, you should definitely lint/format this. This part should be an easy catch.